src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -86,12 +86,12 @@ private void testCreateInstructorAction() throws Exception {
         BackDoor.deleteCourse(demoCourseId);
         BackDoor.deleteInstructor(demoCourseId, instructor.email);
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
```

Comment:
Ok. I am not sure whether this is the best solution (and best design). (You can create your own AppVeyor instance and try run the test case, the retry time for `AdminHomePageUiTest` should not exist in gradle script).

So we still don't get rid of renaming methods (`getInstructor` -> `getInstructorWithRetry` etc). @wkurniawan07 Your inputs here ?